src/components/Banner/ImageCard.css

```diff
@@ -0,0 +1,91 @@
+.image-card {
+  position: relative;
+  width: 100%;
+  overflow: hidden;
+  height: 100%;
+  border-radius: 0;
+}
+
+.card {
+  width: 100%;
+}
+
+.card-image {
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+}
+
+.hover-part {
+  width: 100%;
+  height: 100%;
+}
+
+.hover-top {
+  position: absolute;
+  z-index: 2;
+  top: 5px;
+  right: 15px;
+  display: flex;
+  gap: 10px;
+  list-style: none;
+}
+
+.icon-image {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 40px;
+  height: 40px;
+  border: 0;
```

Comment:
Should we need to specify `border:0` here?

Reply:
other wise a black border is appearing

Reply:
Then keep it like this. We can check this later.

Reply:
ok